### ISSUE TITLE
uWebSocket client immediate disconnects after connection open

### ISSUE THREAD

**Issue by FMarxis** [uWebSockets/uWebSockets#587]
* What operating system are you running?
  * Linux Ubuntu 16.04 LTS
* Are you running any CPU architechute out of the normal?
  * No
* What platform/language are you using?
  * Node.js
* Are you using SSL, if so, how?
  * SSL via µWS or Node.js
* What version of uws are you using?=
  * Release v0.14.4
* What features of µWS are involved in the issue?
  * Client support (WebSocket)
* How does the issue reproduce?
  * All the time
* Can you show me some short code that triggers your issue or any other important information?

First of all would like to request to push uws@0.13.0 to NPM repo because currently this is the only version that worked perfectly for me without breaking.

Confirmed that segfault on WebSocket connection error has been fixed in 0.14.3 and 0.14.4. But a new bug has been introduced in the both versions. Clients would get disconnected by WebSocket Server on connection almost immediately. No errors thrown, not sure if C++ native side is affected as well.

```
function trial() {
    var ws = new WebSocket('wss://192.168.30.117:45267');
    ws.on('error', function error() {
        console.log('Error connecting!');
    });
    ws.on('message', function (message) {
        console.log(message);
    });
    ws.on('open', function (message) {
            console.log("Opened");
        });
    ws.on('close', function (message) {
        console.log("closed");
    });
}
trial();
```

Desired behaviour would be the client sit there listening to messages from the server. Confirmed with Chrome and Firefox browser implementation of WebSocket clients and uws@0.13.0. But in 0.14.3 and 0.14.4, the client connection would open then get closed immediately.

**Comment by kuwkuw** [comment]
I have same issues on WIndows 10 . 
Returning to 0.14.1 fixed this.

**Comment by alexhultman** [comment]
It is a quite small bug that comes from the addition of configurable maxPayload. The bug is not about when you connect (your code works as it should) but if you receive something it will terminate the connection because currently there is an implicit 0 passed as maxPayload for the client group. I will fix this and publish a new release - this is just in the uws.js so it should require a rebuild.

**Comment by alexhultman** [comment]
Fixed in 0.14.5
